app/controllers/update-job-view-model.js

```diff
@@ -0,0 +1,30 @@
+const i18n = require('i18n');
+const progression = require('../models/progression');
+/* eslint-disable no-underscore-dangle */
+
+class UpdateJobViewModel {
+  constructor(accountId, job) {
+    Object.assign(this, job);
+    this.accountId = accountId;
+
+    const ratingOptionList = [5, 4, 3, 2, 1].map(value => ({ value }));
```

Comment:
Maybe worth carrying out a quick manual check to make sure the right ratings end up in the DB.